packages/chopsticks/src/schema/parse.test.ts

```diff
@@ -0,0 +1,60 @@
+import { describe, expect, it } from 'vitest'
+import { readdirSync } from 'fs'
+import _ from 'lodash'
+import path from 'path'
+
+import { configSchema, fetchConfig } from './index.js'
+
+function getAllFiles(dirPath: string) {
+  const files = readdirSync(dirPath)
+  const arrayOfFiles: string[] = []
+  files.forEach(function (file) {
+    arrayOfFiles.push(path.join(dirPath, '/', file))
+  })
+
+  return arrayOfFiles
+}
+
+describe('Existing configs', async () => {
+  const configPaths = getAllFiles(path.join(__dirname, '../../../../configs'))
+  it.each(configPaths.map((p) => ({ name: _.last(p.split('/')), path: p })))(
+    '$name config parsing is correct',
+    async ({ path }) => {
+      const config = await fetchConfig(path)
+      expect(configSchema.parse(config)).toMatchSnapshot()
```

Comment:
this is useful, with one issue that add or modify config will need to regenerate snapshots. maybe the easy way is just not check the output but only check parse is successful